### ISSUE TITLE
[Content styleguide]- Content doesn't align with style guidance in VA.gov content style guide (05.08.1)

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#33867]
#### VFS team name

AFS Education

#### VFS product name

GI Bill Comparison Tool

#### Point of Contact/Reviewers (Practice area)

Randi Hecht (Content)

---

### Platform Issue

Content doesn't align with style guidance in VA.gov style guide

### Issue Details

In the places with running text (Institution comparison, for example) commas are missing in lists, dates, and numbers, and words are capped inconsistently or when they shouldn't be.

### VA.gov Experience Standard

[05.08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References

WCAG SC 3.2.4AA

---

### Platform Recommendation

The team should open a Sitewide Content intake request ticket to have the team do a review of the places in the tool where there is running text, like the Learn more content blocks and the institution comparison text.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket. 

## Platform directions
- Update "Issue Title"
- Link to collab cycle touchpoint meeting epic
- Add to the Feedback pipeline on the Platform Product Support board
- Add your feedback
- Add assignees: VFS PM
- Add labels:
  - VFS team label
  - VA.gov Experience Standard Issue number (CCIssueXX.YY) where XX is the category number and YY is the issue number, per the VA.gov Experience Standards
  - Practice area labels (508/accessibility, IA, content, design) per the VA.gov Experience Standards
  - 508 defect label, if applicable (508-defect-0, 508-defect-1, 508-defect-2, 508-defect-3, 508-defect-4)
  - Launch-blocking label, if applicable

**Comment by mrblanco1** [comment]
This issue has been indirectly resolved with #33866 on 3/14/2022
